templates/adp-backend-template-dotnet/skeleton/test/${{ values.dotnet_solution_name }}.Api.Tests/${{ values.dotnet_solution_name }}.Api.Tests.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>net8.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+
+		<IsPackable>false</IsPackable>
+		<IsTestProject>true</IsTestProject>
+		<!-- Exclude coverage from Core project. Outputs coverage only for Api projects -->
+		<Exclude>[${{ values.dotnet_solution_name }}.Core]*</Exclude>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="AutoFixture" Version="4.18.1" />
+		<PackageReference Include="coverlet.msbuild" Version="6.0.2">
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+			<PrivateAssets>all</PrivateAssets>
+		</PackageReference>
+		<PackageReference Include="FluentAssertions" Version="6.12.0" />
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
+		<PackageReference Include="NSubstitute" Version="5.1.0" />
+		<PackageReference Include="NUnit" Version="4.1.0" />
```

Comment:
Should be using XUnit I think, as its the modern successor to NUnit and is used by microsoft for their testing